src/microbe_stage/SpawnSystem.cs

```diff
@@ -40,6 +40,13 @@ public class SpawnSystem
     [JsonProperty]
     private int maxAliveEntities = 1000;
 
+    /// <summary>
+    ///   This limits the number of things that can be spawned in a single spawn radius.
+    ///   Used to limit items spawning in one single circle when the player doesn't move.
+    /// </summary>
+    [JsonProperty]
+    private int maxEntitiesInSpawnRadius = 15;
```

Comment:
shouldn't this be a constant in the constants class?

Reply:
I kept the same logic as for `maxAliveEntities`, but I'm definitely open to move both to `Constants.cs`

Reply:
I think this is a field because in theory this could change, I think? That's why it is also saved to keep loaded saves consistent.

Reply:
Then I'll keep the same approach here. If this really ought to be changed for some reason I'll leave it to another PR.